sources/osgViewer/Viewer.js

```diff
@@ -404,18 +408,12 @@ Viewer.prototype = MACROUTILS.objectInherit( View.prototype, {
         var self = this;
         var render = function () {
             if ( !self.done() ) {
-
-                var isPresentingVR = self._hmd && self._hmd.isPresenting;
-
-                if ( isPresentingVR )
+                if ( self._hmd && self._hmd.isPresenting )
```

Comment:
The signatures are a bit different (the vr requestAnimation doesn't take a html 2nd element in argument for example).
Also, if I'm not mistaken, isPresenting must be read at each frame as it's the only way to know if we have to render in vr or not.

What's the issue with the conditional? It takes 0 perf and should be completely transparent in non vr.